setup/vagrant/provision.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+
+cd /opt/redash/current
+cp /opt/redash/.env /opt/redash/current
+cd /opt/redash/current/rd_ui
+bower install
+cd /opt/redash/current
+
+#install requirements
+sudo pip install -r /opt/redash/current/requirements_dev.txt
+sudo pip install -r /opt/redash/current/requirements.txt
+sudo pip install pymongo==3.2.1
```

Comment:
It's better to run `sudo pip install -r /opt/redash/current/requirements_all_ds.txt` here.